jumpscale/sals/nginx/templates/nginx.conf

```diff
@@ -32,7 +32,7 @@ http {
     proxy_connect_timeout       1d;
     proxy_send_timeout          1d;
     proxy_read_timeout          1d;
-
+    proxy_buffering off;
```

Comment:
This can be moved on location level, we can add a new options for buffering in `Location`.

Reply:
This will avoid setting it globally in the main nginx.conf

Reply:
Also, this problem cannot be resolved by setting correct `proxy_buffer_size` and `proxy_buffers` (also in the location level)?

http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffers
http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffer_size